src/content/docs/using-new-relic/welcome-new-relic/get-started/glossary.mdx

```diff
@@ -26,7 +26,9 @@ A glossary of common terminology you may encounter.
     id="account-dropdown"
     title="account dropdown"
   >
-    When you're logged into New Relic, the account dropdown menu displays your login name and is located at the top right of the UI. Clicking this gives you access to various [account-related abilities](/docs/accounts).
+    When you're logged into New Relic, the account dropdown menu displays your login name and is located at the top left of the UI. To switch accounts, click the account switcher dropdown in the top left. 
```

Comment:
> When you're logged into New Relic, the account dropdown menu displays your login name and is located at the top left of the UI. 

Is this true? For me it shows "All accounts".

Reply:
Oh yeah, good call. Does it show your login name if you only have one account? I'm trying to verify that.

Reply:
It says All accounts even when you're just on one account. It's just greyed out.